backend/schema/heat_pump_dryer.ts

```diff
@@ -0,0 +1,43 @@
+import { FromSchema } from "json-schema-to-ts";
+import { metadataProperties, requiredMetadata } from "./metadata";
+import { coreProperties, requiredCore } from "./core";
+
+const noise = ["Low", "Medium", "High"];
+
+export const heatPumpDryerProperties = {
+  capacity: {
+    type: "number",
+  },
+  noise: {
+    type: "string",
+    enum: noise,
+  },
```

Comment:
I think we can remove "noise" – unless you've seen it somewhere, that's not how dryers would indicate their volume – they'd give decibels. And that's already part of the core properties.